.rubocop_todo.yml

```diff
@@ -373,14 +365,13 @@ Style/StringConcatenation:
     - 'app/jobs/modsulator_job.rb'
     - 'app/models/concerns/permitted_queries.rb'
     - 'app/models/report.rb'
-    - 'app/models/track_sheet.rb'
     - 'app/models/user.rb'
     - 'app/search_builders/argo/access_controls_enforcement.rb'
     - 'spec/jobs/modsulator_job_spec.rb'
 
-# Offense count: 376
-# This cop supports safe auto-correction (--auto-correct).
-# Configuration parameters: AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
+# Offense count: 386
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, AllowedPatterns, IgnoredPatterns.
 # URISchemes: http, https
 Layout/LineLength:
-  Max: 178
+  Max: 2192
```

Comment:
This doesn't seem good.  Can you move the exception to the file that has this tremendously long line?

Reply:
I'm fixing the linelength instead.